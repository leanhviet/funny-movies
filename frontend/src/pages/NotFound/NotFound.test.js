// Libs
import React from 'react'
import { shallow } from 'enzyme'

// Components
import { NotFound } from './index'

describe('NotFound page', () => {
  const wrapper = shallow(<NotFound />)

  it('should be defined NotFound page', () => {
    expect(wrapper).toBeDefined()
  })

  // Testing Snapshot
  it('should be render snapshot NotFound page', () => {
    expect(wrapper).toMatchSnapshot()
  })
})
