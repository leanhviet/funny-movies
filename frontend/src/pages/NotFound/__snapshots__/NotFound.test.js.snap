// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound page should be render snapshot NotFound page 1`] = `
<div
  className="container"
>
  <div
    className="row"
  >
    <div
      className="col-md-12"
    >
      <div
        className="notfound"
      >
        <h1>
          Oops!
        </h1>
        <h2>
          404 Not Found
        </h2>
        <div
          className="notfound__details"
        >
          <p>
            Sorry, an error has occured, Requested page not found!
          </p>
        </div>
        <div
          className="notfound__actions"
        >
          <a
            className="btn btn-primary btn-lg"
            href="/"
          >
            <span
              className="glyphicon glyphicon-home"
            />
            Take Me Home
          </a>
        </div>
      </div>
    </div>
  </div>
</div>
`;
